independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2444 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenNEM",

```

Comment:
@paulculmsee We usually name the connector folders as per their title. Can you please rename the connector with its title `"OpenNEM"`.